src/ensembl/src/shared/helpers/regionFormatter.ts

```diff
@@ -0,0 +1,10 @@
+import { getCommaSeparatedNumber } from 'src/shared/helpers/numberFormatter';
+
+import { EnsObjectLocation } from 'src/ens-object/ensObjectTypes';
+
+export const getFormattedLocation = (location: EnsObjectLocation) => {
+  const start = getCommaSeparatedNumber(location.start);
+  const end = getCommaSeparatedNumber(location.end);
+
+  return `${location.chromosome}:${start}-${end}`;
+};
```

Comment:
Can this function go into `browserHelper.ts`?

Reply:
Is this browser-specific though?

Won't we have an EntityViewer which might need to display coordinates of an Entity?

Reply:
True. Let's keep this separate then.